docs/core/tools/sdk-errors/netsdk1195.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "NETSDK1195: Trimming, or code compatibility analysis for trimming, single-file deployment, or ahead-of-time compilation is not supported for the target framework."
+description: How to resolve compatibility problems with features that rely on the ILLink pack.
+ms.topic: error-reference
+ms.date: 06/05/2023
+f1_keywords:
+- NETSDK1195
+---
+# NETSDK1195: Trimming, or code compatibility analysis for trimming, single-file deployment, or ahead-of-time compilation is not supported for the target framework.
+
+NETSDK1195 indicates that you're using a feature of the SDK which is not available for the selected target framework. The following features depend on the ILLink pack, which is only available when targeting `netcoreapp3.0` and above:
+
+- Trimming (via `PublishTrimmed` or `PublishAot`)
+- Trim analysis (via `PublishTrimmed`, `PublishAot`, `IsTrimmable`, `IsAotCompatible`, or `EnableTrimAnalyzer`)
+- Single-file analysis (via `PublishSingleFile` or `EnableSingleFileAnalyzer`)
+- Ahead-of-time compilation analysis (via `PublishAot` or `EnableAotAnalyzer`)
```

Comment:
This might give the false impression that AOT analysis will work on `netcoreapp3.0` - maybe we should mention that this is only really supported on `net7.0` and above.

Which leads me to another question: Maybe we should issue this warning when somebody targets `net6.0` and sets `PublishAot` for example.  Or maybe we should have a similar warning for AOT. It's basically the same story as for trimming and `netstandard2.0`. It won't do anything but it will provide false "hope" that the app doesn't have AOT incompatibilities.